provider/aws/aws.go

```diff
@@ -500,22 +529,12 @@ func (p *AWSProvider) newChange(action string, ep *endpoint.Endpoint, recordsCac
 		if val, ok := ep.Labels[endpoint.DualstackLabelKey]; ok {
 			dualstack = val == "true"
 		}
-
 		change.ResourceRecordSet.Type = aws.String(route53.RRTypeA)
```

Comment:
What about CNAME aliases?

Reply:
Hm this was part wasn't changed in this pull request. I assume CNAME alias records don't work (and didn't work before either). I guess that would be a new feature request.

Reply:
You're right. This PR's scope is already quite large, and has been pending for a long time. Would love to see it merged so we can add CNAME alias support.